src/Generator/Helper.cs

```diff
@@ -22,4 +22,20 @@ internal static bool IsObsolete(this DTNamedEntityInfo entity)
         AccessModifier.PrivateProtected => "private protected",
         _ => throw new Exception($"Invalid value for AccessModifier: {modifier}"),
     };
+
+    public static int IndexOf<T>(this IReadOnlyList<T> self, T elementToFind)
```

Comment:
I'm failing to figure out what his method is needed for?

Reply:
Because the IReadOnlyInterface doesn't have an IndexOf method, although I agree, I don't see a usage for it.

Reply:
This is being used in the WriteContent method in EnumPropEntity class.

Reply:
So why wasn't needed before? Did the new package change the behavior of `WriteContent()`?

Reply:
Previously it was just List<DTEnumValueInfo> but after Package change the type is IReadOnlyList<DTEnumValueInfo>